cluster/config-defaults.yaml

```diff
@@ -343,14 +343,17 @@ teapot_admission_controller_daemonset_reserved_memory: "64Gi"
 teapot_admission_controller_validate_application_label: "true"
 teapot_admission_controller_validate_base_images: "true"
 teapot_admission_controller_validate_pod_images: "true"
+teapot_admission_controller_validate_pod_images_soft_fail_namespaces: "^pierone$"
```

Comment:
Should this be set for all clusters or only for the one where pierone runs?

Reply:
Only for one cluster. Thanks! 🤦